[2023-09-11T04:54:56.981+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test01.test02 manual__2023-09-11T04:54:49.513476+00:00 [queued]>
[2023-09-11T04:54:56.986+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test01.test02 manual__2023-09-11T04:54:49.513476+00:00 [queued]>
[2023-09-11T04:54:56.986+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-09-11T04:54:56.992+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): test02> on 2023-09-11 04:54:49.513476+00:00
[2023-09-11T04:54:56.996+0000] {standard_task_runner.py:57} INFO - Started process 501 to run task
[2023-09-11T04:54:57.000+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'test01', 'test02', 'manual__2023-09-11T04:54:49.513476+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/Orders_realPaidPrice.py', '--cfg-path', '/tmp/tmpbbl9nz0q']
[2023-09-11T04:54:57.001+0000] {standard_task_runner.py:85} INFO - Job 58: Subtask test02
[2023-09-11T04:54:57.035+0000] {task_command.py:415} INFO - Running <TaskInstance: test01.test02 manual__2023-09-11T04:54:49.513476+00:00 [running]> on host 065aab486aca
[2023-09-11T04:54:57.088+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test01' AIRFLOW_CTX_TASK_ID='test02' AIRFLOW_CTX_EXECUTION_DATE='2023-09-11T04:54:49.513476+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-11T04:54:49.513476+00:00'
[2023-09-11T04:54:57.148+0000] {logging_mixin.py:151} INFO - s3 bucket connected!
[2023-09-11T04:54:57.263+0000] {logging_mixin.py:151} INFO - 데이터가 S3 버킷 wingeatdata의 practice/2023-9-8-5.csv 위치에 저장되었습니다.
[2023-09-11T04:54:57.264+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-11T04:54:57.273+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test01, task_id=test02, execution_date=20230911T045449, start_date=20230911T045456, end_date=20230911T045457
[2023-09-11T04:54:57.294+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-11T04:54:57.308+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
